vue.config.js

```diff
@@ -39,4 +39,21 @@ module.exports = {
       },
     },
   },
+
+  devServer: {
+    proxy: {
+      '/game': {
+        target: 'http://localhost:1337',
+        changeOrigin: true,
+      },
+      '/user': {
+        target: 'http://localhost:1337',
+        changeOrigin: true,
+      },
+      '/test': {
+        target: 'http://localhost:1337',
+        changeOrigin: true,
+      },
+    },
+  },
```

Comment:
proxy vue requests as if they came from vue for these routes with a reverse proxy

Reply:
siiiiick